### PR TITLE
Task_DEV-2

### PR DESCRIPTION
**Pull request by MaryVazhnik**

### CODE REVIEW

File: DEV-2/DEV_2/Selection.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Text;
+
+namespace DEV_2
+{
+    /// <summary>
+    ///Class for deleting all odd symbols 
+    /// </summary>
+    class Selection
```
Comment (IgorKheidorov) [inline DEV-2/DEV_2/Selection.cs]: Bad name

File: DEV-2/DEV_2/Selection.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Text;
+
+namespace DEV_2
+{
+    /// <summary>
+    ///Class for deleting all odd symbols
```
Comment (IgorKheidorov) [inline DEV-2/DEV_2/Selection.cs]: The task i not to delete, the task is to form new string from other

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: DEV-2/DEV_2/SequenceBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Text;
+
+namespace DEV_2
+{
+    /// <summary>
+    ///Class creates a new string from the previous
```
Comment (AlexanderSoroka) [inline DEV-2/DEV_2/SequenceBuilder.cs:7]: missed space after ///

File: DEV-2/DEV_2/SequenceBuilder.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        static void Main(string[] args)
+        {
+           Console.WriteLine("Write a string");
+           String s = Console.ReadLine(); 
+           StringBuilder sb = new StringBuilder();
+           for (int i = 0; i < s.Length; i+=2)
+           {
+              sb.Append(s[i]);
+           }
+           Console.WriteLine("Symbols with even indices: "+sb);
```
Comment (AlexanderSoroka) [inline DEV-2/DEV_2/SequenceBuilder.cs:23]: codestyle violation

File: DEV-2/DEV_2/SequenceBuilder.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    /// </summary>
+    class SequenceBuilder
+    {
+        /// <summary>
+        /// Entry Point.
+        /// </summary>
+        static void Main(string[] args)
+        {
+           Console.WriteLine("Write a string");
+           String s = Console.ReadLine();
```
Comment (AlexanderSoroka) [inline DEV-2/DEV_2/SequenceBuilder.cs:17]: bad variable name

**Review by AlexanderSoroka — APPROVED** [review]

**Pull request merged**